deploy/etc/deployments/zgc_cluster_config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: zgc-cluster-config
+  labels:
+    name: zgc-cluster-config
+data:
+  cidr: 10.0.0.0/24
```

Comment:
Add newline at end of file
add MAC_PRIVATE here as well

Reply:
fixed